app/components/pool/details/pool-configuration.html

```diff
@@ -57,7 +57,7 @@
             <bl-text-property label="User identity" [value]="decorator.userIdentitySummary"></bl-text-property>
```

Comment:
I would wrap all the start task properties into a div or template and have a ngIf startTask instead